src/notifications/templates/notifications/email/series_events_generated.html

```
@@ -0,0 +1,40 @@
+{% load i18n %}<!DOCTYPE html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)